src/components/auth/login.js

```diff
@@ -128,6 +133,7 @@ class LoginPage extends React.Component {
                 </div>
 
                 <div className="BtnContainer">
+                  <Link to="/register">Already have an account?</Link>
```

Comment:
Added a way to get to register page.

Reply:
Instead of >Already have an account?</Link> Shouldn't that be >Don't have an account?</Link> Since you are redirecting them to Register - to make an account? Just making sure!

Reply:
All done in [this commit](https://github.com/Lambda-School-Labs/Groa-fe/pull/15/commits/1917c479fbd14e60b8a47a344a479e378f3875d5)